sapicore/tests/data/test_drift.py

```diff
@@ -11,10 +11,12 @@ class TestDrift:
     @pytest.mark.unit
     def test_drift_processing(self):
         # calling a dataset invokes its load() method and returns a self reference.
-        data = DriftDataset(root=os.path.join(TEST_ROOT, "drift")).load()
+        try:
+            DriftDataset(root=os.path.join(TEST_ROOT, "drift")).load()
 
-        # demonstrate logical selection of sample indices based on label values.
-        data.select(["chemical.isin([1])", "batch==10"])
+        except ConnectionError:
+            # package tests should NOT fail when a particular hardcoded archive server is down.
+            pass
```

Comment:
```suggestion
            pytest.mark.skip("Could not download drift dataset")
```

Btw, overall, what's the point of this test? You seemed to have removed the `select` so it only downloads stuff.

Reply:
The idea was to add a separate method for testing `select` once we implement the compromise we discussed in the base data class (saved for a separate PR, as requested). `_download` and `load` are substantial and should be tested for the drift set--it's used instructively in the tutorials to help users get started; if it fails, I'd like to know it and investigate.

Generally, we should aim for full coverage within reason--I'd like to expand the test suite when I have downtime.

Reply:
Oh, ok so it's kinda a placeholder for now until you can re-expand the test!? That's fine! 

Are you ok adding the `skip` line I added? Because it'd better if when a test is skipped that it's visible in the results.